_posts/en/posts/2016-01-24-clarification.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Clarifying Communications
+name: clarifying-communications
+type: post
+permalink: /en/2015/01/24/clarification
+share: true
+---
+The Bitcoin Core team would like to address some recent misunderstandings about the relationship between Bitcoin Core and public venues for information and discussion about Bitcoin itself.
+
+We encourage the Bitcoin community to civilly discuss and critique any and all changes to the system. However, we also recognize that venues have specific purposes and policies that must be respected. Our wider community can become extremely excited and heated when it comes to discussing Bitcoin. At the same time, there is universal agreement to not engage in brigading, to reject participation in denial-of-service attacks, and to assume good faith in absence of reason to believe otherwise.
```

Comment:
The following inner (crossed) part is very fluffy in my opinion and could be removed completely, to focus on the actual message, without losing much information:

> We encourage the Bitcoin community to civilly discuss and critique any and all changes to the system. ~~However, we also recognize that venues have specific purposes and policies that must be respected. Our wider community can become extremely excited and heated when it comes to discussing Bitcoin. At the same time, t~~ **There is universal agreement*** to not engage in brigading, to reject participation in denial-of-service attacks, and to assume good faith in absence of reason to believe otherwise.

\*Who is agreeing here? The authors of the website? The core devs? In my opinion this should be named more explicitly with something like "there is universal agreement amongst ... this and that group".

Furthermore, the quoted part might be moved to the bottom, or near the bottom.